src/Writers/HTMLWriter.jl

```diff
@@ -667,6 +669,7 @@ if isdefined(Base.Markdown, :Admonition)
             div[".admonition-text"](mdconvert(a.content, a))
         )
     end
+    mdconvert(c::Markdown.Code, parent::Markdown.Admonition) = mdconvert_codeblock(c, parent)
 end
```

Comment:
The other toplevel blocks should also have this change, not just `Admonition`s and `MD`s, correct? `BlockQuote`, `Footnote`, and `List` elements as well?